<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>(C) 2004 Peter Simonsson</author>
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
</comment>
 <class>JoinChannelUI</class>
 <widget class="QWidget" name="JoinChannelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>94</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="channelLabel">
     <property name="text">
      <string>C&amp;hannel:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>channelCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KHistoryComboBox" name="channelCombo">
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
     <property name="contextMenuEnabled" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>&amp;Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>passwordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Connection:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="networkNameCombo"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>networkNameCombo</tabstop>
  <tabstop>channelCombo</tabstop>
  <tabstop>passwordEdit</tabstop>
 </tabstops>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
